src/Arcus.Scripting.ApiManagement/Scripts/Create-AzApiManagementUserAccountsFromConfig.ps1

```diff
@@ -0,0 +1,215 @@
+            },
+            "sendNotification": {
+              "type": "boolean"
+            },
+            "confirmationType": {
+              "type": "string",
+              "enum": ["signup", "invite"]
+            },
+            "note": {
+              "type": "string"
+            },
+            "groups": {
+              "type": "array",
+              "prefixItems": [
+                {
+                  "type": "object",
+                  "additionalProperties": false,
+                  "properties": {
+                    "id": {
+                      "type": "string"
+                    },
+                    "displayName": {
+                      "type": "string"
+                    },
+                    "description": {
+                      "type": "string"
+                    }
+                  },
+                  "required": [
+                    "id",
+                    "displayName"
+                  ]
+                }
+              ]
+            },
+            "subscriptions": {
+              "type": "array",
+              "prefixItems": [
+                {
+                  "type": "object",
+                  "additionalProperties": false,
+                  "properties": {
+                    "id": {
+                      "type": "string"
+                    },
+                    "displayName": {
+                      "type": "string"
+                    },
+                    "scope": {
+                      "type": "string"
+                    },
+                    "allowTracing": {
+                      "type": "boolean"
+                    }
+                  },
+                  "required": [
+                    "id",
+                    "displayName",
+                    "scope",
+                    "allowTracing"
+                  ]
+                }
+              ]
+            }
+          },
+          "required": [
+            "firstName",
+            "lastName",
+            "mailAddress",
+            "sendNotification",
+            "confirmationType"
+          ]
+        }
+      ]
+    }
+'@
+
+if (-not (Get-Content -Path $ConfigurationFile -Raw | Test-Json -Schema $schema -ErrorAction SilentlyContinue)) {
+    throw "Cannot apply user configuration based on JSON configuration file because the file does not contain a valid JSON configuration file."
+}
+
+$apim = Get-AzApiManagement -ResourceGroupName $ResourceGroupName -Name $ServiceName
+if ($apim -eq $null) {
+    throw "Unable to find the Azure API Management service '$ServiceName' in resource group '$ResourceGroupName'"
+}
+$apimContext = New-AzApiManagementContext -ResourceGroupName $ResourceGroupName -ServiceName $ServiceName
+
+if ($SubscriptionId -eq "" -or $AccessToken -eq "") {
+    # Request accessToken in case the script contains no records
+    $token = Get-AzCachedAccessToken
+
+    $AccessToken = $token.AccessToken
+    $SubscriptionId = $token.SubscriptionId
+}
+
+try
+{
```

Comment:
Should this bracket also be on the `try`? Like `try {`?
Now I'm nitpicking 😄

Reply:
As a fellow nitpicker I love it 😉 
Fixed in [`2a66263` (#422)](https://github.com/arcus-azure/arcus.scripting/pull/422/commits/2a66263a37250b68cb99fef6eb08cc7e08e1d3d3)